onnxruntime/test/providers/qnn/layer_norm_test.cc

```diff
@@ -158,7 +158,20 @@ TEST_F(QnnHTPBackendTests, LayerNorm1D_LastAxis_StaticScale_AU16_WU8) {
 }
 
 // Test accuracy of 8-bit QDQ LayerNorm with a dynamic scale input.
-TEST_F(QnnHTPBackendTests, LayerNorm1D_LastAxis_DynamicScale) {
+//
+// TODO(adrianlizarraga): Fails to finalize with QNN SDK 2.22.
+// Verbose logs:
+// Starting stage: Graph Transformations and Optimizations
+// C:\...\QNN\HTP\HTP\src\hexagon\prepare\graph_prepare.cc:203:ERROR:could not create op: q::flat_to_vtcm
+// C:\...\QNN\HTP\HTP\src\hexagon\prepare\graph_prepare.cc:1187:ERROR:Op 0x102800000013 preparation failed with err:-1
+// Completed stage: Graph Transformations and Optimizations (6247 us)
+// QnnDsp <E> "node_token_15" generated: could not create op
+// QnnDsp <E> RouterWindows graph prepare failed 12
+// QnnDsp <E> Failed to finalize graph (id: 1) with err 1002
+// QnnDsp <V> Wake up free backend 1 thread(s)
+// QnnDsp <I> QnnGraph_finalize done. status 0x3ea
+// Failed to finalize QNN graph.
+TEST_F(QnnHTPBackendTests, DISABLED_LayerNorm1D_LastAxis_DynamicScale) {
```

Comment:
@HectorSVC @jywu-msft FYI, This LayerNorm with dynamic scale input (input[1]) fails to finalize on the HTP backend.